tests/test_anonymization_overview.py

```
@@ -0,0 +1,314 @@
+"""
+Unit tests for anonymization overview API endpoints.
+Tests the /api/anonymization/items/overview/ endpoint that returns
+FileItem interface data for videos and PDFs.
+"""
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)